tests/test_category.py

```diff
@@ -0,0 +1,12 @@
+from saleor.product.filters import ProductCategoryFilter
+
+
+def test(product_type, categories_tree):
```

Comment:
We could use a more meaningful name that describes what this test is actually testing ;)